packages/frontend/src/Patreon/index.tsx

```diff
@@ -6,12 +6,14 @@ import { Button, Dialog, DialogActions } from '@material-ui/core'
 import { githubLogo, patreonLogo } from '../common/images'
 import styles from './styles.module.scss'
 
+const isOctober = DateTime.now().month === 10
+
 const Patreon = () => {
   const [open, setOpen] = useState<boolean>(false)
 
   useEffect(() => {
     if (!document.cookie.includes('patreonpop')) {
-      const threeDays = DateTime.utc().plus({ days: 3 })
+      const threeDays = DateTime.utc().plus({ days: isOctober ? 1 : 3 })
```

Comment:
change the name of the variable from `threeDays` to something like `reDisplayTime` (not the best name, I know), because in october it isn't 3 days, that would just make a bit more sense

Reply:
True, it was just a quick change.